lib/Doctrine/ruleset.xml

```diff
@@ -100,12 +100,16 @@
                 name="forbiddenAnnotations"
                 type="array"
                 value="
+                    @api,
```

Comment:
`@api` **can** be useful when dealing with BC promises or the lack thereof, but I don't think we use it for now, so it's probably ok to include it.

Reply:
If it is `public` and it is not marked `@internal` then it is already API :P

Reply:
Don't make it so simple!

Reply:
It is that simple :P

Reply:
Agreed, `@api` adds maintenance overhead. Any public or non-final protected is typically public API _by design_. At least it motivates devs to think about API & what should be exposed and what not (looking at you SF). 👍